tests/simulation/params/test_validators_params.py

```diff
@@ -74,6 +93,29 @@ def test_consistency_wall_function_validator(
         )
 
 
+def test_ddes_wall_function_validator(
+    volume_output_with_SA_DDES,
+    volume_output_with_kOmega_DDES,
+    fluid_model_with_DDES,
+    fluid_model,
+):
+    # Valid simulation params
+    with SI_unit_system:
+        params = SimulationParams(
+            models=[fluid_model_with_DDES], outputs=[volume_output_with_SA_DDES]
+        )
+
+    assert params
+
+    message = "kOmegaSST_DDES output can only be specified with kOmegaSST turbulence model and DDES turned on."
+
+    # Invalid simulation params (wrong output type)
+    with SI_unit_system, pytest.raises(ValueError, match=re.escape(message)):
+        _ = SimulationParams(
```

Comment:
```suggestion
        SimulationParams(
```

Reply:
Could you also add a test when DDES is disabled?

Reply:
Added in [0a08e8b](https://github.com/flexcompute/Flow360/pull/440/commits/0a08e8bc2a82aa448bb7c4c1831f355ffe2266f0)